Source/Core/DolphinQt/GameList/GameListModel.cpp

```diff
@@ -135,9 +135,20 @@ QVariant GameListModel::data(const QModelIndex& index, int role) const
     break;
   case COL_SIZE:
     if (role == Qt::DisplayRole)
-      return QString::fromStdString(UICommon::FormatSize(game.GetFileSize()));
+    {
+      std::string str = UICommon::FormatSize(game.GetFileSize());
+
+      // Add asterisk to size of compressed files.
+      if (game.GetFileSize() != game.GetVolumeSize())
+        str += '*';
+
+      return QString::fromStdString(str);
+    }
     if (role == Qt::InitialSortOrderRole)
       return static_cast<quint64>(game.GetFileSize());
+    if (role == Qt::ToolTipRole && game.GetFileSize() != game.GetVolumeSize())
+      return QString(tr("* Uncompressed size: ") +
```

Comment:
Curlies please, since it spans over multiple lines.

Also: Should this get a `i18n` comment indicating that the asterisk in the message refers to the asterisk shown in the column and should be left in a translation?